docs/src/pages/[platform]/components/radiogroupfield/react.mdx

```diff
@@ -146,9 +146,16 @@ Use the `hasError` and `errorMessage` props to mark a RadioGroupField as having
   </ExampleCode>
 </Example>
 
-### Accessibility / Label behavior
+### Accessibility / Legend behavior
 
-<Fragment>{() => import('./../shared/formFieldAccessibility.mdx')}</Fragment>
+The form primitives are accessible by default. A matching `legend` HTML element will be connected to the form control -- simply provide a `legend` prop with a `string` or `ReactNode`. If no `id` is provided, one will be automatically generated and connected to both `legend` and form control elements.
+
+#### How can I visually hide the legend while maintaining accessibility?
+
+In some applications it may be desirable to hide the legend when the field purpose is visually apparent such as a search field with button.
+The `legendHidden` prop can be used to visually hide the legend while still keeping it visible for screenreaders.
+
+UX Tip: refrain hiding legend if doing so hides important information needed by the user to fill out the field.
```

Comment:
```suggestion
#### How can I visually hide the legend or labels while maintaining accessibility?

In some applications it may be desirable to hide the legend when the field purpose is visually apparent such as a search field with button.
The `legendHidden` prop can be used on `RadioGroupField` to visually hide the `<legend>` while still keeping it available to screenreaders. Similarily, the `labelHidden` prop can be used to accessibly hide a `<Radio>` element's `<label>`

UX Tip: refrain hiding legends and labels if doing so hides important information needed by the user to fill out the field.
```
Some more copy suggestions for this part since we have legend's and labels now.